baggers.js

```diff
@@ -14357,12 +14417,24 @@ var data = {
       lat: 45.764043,
       lng: 4.835659,
     },
+    Luxembourg: {
+      name: 'Luxembourg',
+      ville_img: 'img/villes/BBL_Luxembourg.png',
+      lat: 49.61167,
+      lng: 6.13,
+    },
     Marseille: {
       name: 'Marseille',
       ville_img: 'img/villes/BBL.jpg',
       lat: 43.3,
       lng: 5.4,
     },
+    Metz: {
+      name: 'Metz',
+      ville_img: 'img/villes/BBL_Metz.png',
```

Comment:
You need to add this file in https://github.com/brownbaglunch/BrownBagLunch/tree/gh-pages/img/villes

Reply:
I have just done it.
Thanks for your time and the site as well ;-)

Reply:
Great! thanks